g2p/mappings/langs/crl/crl_equiv.json

```diff
@@ -1,906 +1,916 @@
 [
     {
-        "in": "ᐧᐋ",
-        "out": "ᐙ"
+        "in": "â",
+        "out": "aa"
     },
     {
-        "roman": "thwe",
-        "in": "ᐧᕧ",
-        "out": "ᕨ"
+        "comment": "e is not normally lengthened, so ê goes to e only, not ee",
```

Comment:
I like this! I can imagine we'll get a combination of `comment` and `note` and `notes` or maybe others though. Do you think we should standardize this somehow?

Reply:
Yeah, I agree it would be a good idea to choose a standard name for the note or comment field. Do we have any others elsewhere that could serve as examples to mimic?

Reply:
For what it's worth, we also use `comment` in `iku/iku_sro_to_ipa.json`. I found `notes` in two `config.yaml`, but not in a mapping per se.